src/MooseSoft.Azure.ServiceBus/MooseSoft.Azure.ServiceBus.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <LangVersion>latest</LangVersion>
-    <Version>1.0.1</Version>
+    <Version>1.0.2</Version>
```

Comment:
Modifying param types of existing ctors without support backwards compatibility is a breaking change.  Therefore, the version needs to reflect that or add support for backwards compatible ctors.

Reply:
added the old constructors back for backward compatibility.

Reply:
Added the old constructors back for backward compatibility